src/core/symbolFactories.ts

```diff
@@ -9,36 +9,71 @@ import * as d3 from "d3";
  * A SymbolFactory is a function that takes in a symbolSize which is the edge length of the render area
  * and returns a string representing the 'd' attribute of the resultant 'path' element
  */
-export type SymbolFactory = (symbolSize: number) => string;
+export type SymbolFactory = (symbolSize: number, context?: CanvasRenderingContext2D) => string;
```

Comment:
i was confused by the same, it looks like the logic is there to wrap around the size from the accessor so that it's in pixel space for the user - i'll look into reworking the whole thing